test/identityFactory.js

```diff
@@ -54,9 +54,9 @@ contract('IdentityFactory', (accounts) => {
     proxy = Proxy.at(proxyAddress)
     recoverableController = RecoverableController.at(recoverableControllerAddress)
     recoveryQuorum = RecoveryQuorum.at(recoveryQuorumAddress)
-    await compareCode(proxyAddress, deployedProxy.address)
-    await compareCode(recoverableControllerAddress, deployedRecoverableController.address)
-    await compareCode(recoveryQuorumAddress, deployedRecoveryQuorum.address)
+    //await compareCode(proxyAddress, deployedProxy.address)
+    //await compareCode(recoverableControllerAddress, deployedRecoverableController.address)
+    //await compareCode(recoveryQuorumAddress, deployedRecoveryQuorum.address)
```

Comment:
Checked out this branch ran the tests with the compareCode -> failed
Did a clean clone of the repo, ran tests with compareCode -> worked fine
Truffle must be storing some kind of cache or something, this is amazinigly strange.